uptycs/models.go

```diff
@@ -119,3 +119,138 @@ type ObjectGroup struct {
 	RetentionDays    int          `tfsdk:"retention_days"`
 	Destinations     types.List   `tfsdk:"destinations"`
 }
+
+type TagRule struct {
```

Comment:
nit: can we possibly alphabetize this file

Reply:
If there's a good way to do that automatically I'd totally be on board to prevent drift (and maybe add a blocking lint check)